benchmarks/backend_request_func.py

```diff
@@ -46,6 +46,7 @@ class RequestFuncOutput:
     tpot: float = 0.0  # avg next-token latencies
     prompt_len: int = 0
     error: str = ""
+    prompt: Optional[str] = None
```

Comment:
```suggestion
    prompt: str = ""
```

nit: 만약 어딘가에서 잘못되어 이 값이 빈 값으로 유지되는 경우, 뒤쪽의 파이프라인이 깨지는 것을 방지할 겸, 그리고 입력당 공간을 조금이나마 아낄 겸 `null` 보단 `""` 을 기본값으로 사용하는 것은 어떨까요?